packages/runtime-dom/src/nodeOps.ts

```diff
@@ -1,4 +1,41 @@
+import { warn } from '@vue/runtime-core'
 import type { RendererOptions } from '@vue/runtime-core'
+import type {
+  TrustedHTML,
+  TrustedTypePolicy,
+  TrustedTypesWindow,
+} from 'trusted-types/lib'
+
+let policy: TrustedTypePolicy | undefined = undefined
+function getPolicy(): TrustedTypePolicy | undefined {
+  const ttWindow = window as unknown as TrustedTypesWindow
+  if (ttWindow.trustedTypes && !policy) {
+    try {
+      policy = ttWindow.trustedTypes.createPolicy('vue', {
+        createHTML: val => val,
+        createScript: val => val,
+        createScriptURL: val => val,
```

Comment:
Perhaps this policy should just have the createHTML function defined that way it will error when used for the wrong trusted type. Generally it's a good idea to only allow what's necessary inside the policy.

Reply:
Yeah, that would be better. But I'm still struggling to make TypeScript happy here😂 I'll try later.